### DISCUSSION TITLE
Possibility to open related (child) items

### DISCUSSION THREAD

**Discussion by bvandenberg** [directus/directus#3704] [Feature Requests]
When working with nested data it's nice if you can navigate to the child-item you're looking for by starting at the top-level item. In the example shown below I'd like to be able to directly open the _Screen_ item from the _Interactive_ item that's currently opened.

![image](https://user-images.githubusercontent.com/5312938/104718632-76023b00-572b-11eb-998f-d811201d1f1c.png)

**Comment by benhaynes** [comment]
Hey @bvandenberg — thanks for sharing this. An "Navigate to Item" option would be nice here. Also, we could look into a "Tree View" Layout for browsing these relational items without having to go into the parent at all! I think we have a Feature Request for that already... but seems like a nice one to have.

**Reply by kasparpalgi** [reply]
Do you have somewhere option to vote feature requests? Tree view from us +1 for sure:)

**Comment by bvandenberg** [comment]
Perfect!

**Comment by TomS-** [comment]
Has the view used in this screenshot been removed? I've been unable to replicate it - it looks really great.

**Reply by benhaynes** [reply]
The "Interactive Screens" one? That was the old O2M interface... which is now a repeater style (instead of a table). Would be open to having a table version too. Not sure if that should be in Core or an Extension though (or an Interface Option toggle)... since they're so similar.

**Reply by TomS-** [reply]
For me, it's great to provide context to the fields. I know you could do Company Name: [company_name_field] / Client Name: [client_name_field] but it can get quite long rather than having it list above in a table. I'm unsure how often people would like to use that sort of layout though.

**Reply by orydman-beep** [reply]
@benhaynes that table-like layout is one of very few things we miss in the system. It would be highly appreciated to have it for any relational type of field. For a user storing massive amounts of relationships that type of table gives such an organized feeling to the layout. But surely we can build it as an extension as you suggest. If it could also be with inline edit the feeling would be amazing. The main advantage is you can have more than 2 columns in the same row of the main form.

**Reply by benhaynes** [reply]
@orydman-beep — Not high on the current docket, but we can review an external PR or discuss it being sponsored.

**Comment by rijkvanzanten** [comment]
This has since been added!

**Reply by italy-dude** [reply]
Hi, can you please post a guide on how to use it? I want to select a city by navigating a hierarchy of Continent --> Nation --> City, I am stuck at how to do it. Thank you
